docs/jwst/pipeline/main.rst

```diff
@@ -229,15 +204,94 @@ Control (WFS&C) use a dedicated processing flow through the pipeline stages.
  - Stage 1: WFS&C exposures use the same :ref:`calwebb_detector1 <calwebb_detector1>`
    pipeline processing and steps as regular images.
 
- - Stage 2: The ASN generator identifies WFS&C exposures as those with the
-   string "WFSC" within their "VISITYPE" keyword value. The resulting ASN
-   uses the :ref:`calwebb_wfs-image2 <calwebb_wfs-image2>` pipeline for stage 2
-   processing.
-   This pipeline configuration is identical to :ref:`calwebb_image2 <calwebb_image2>`
-   with the omission of the :ref:`resample <resample_step>` step.
+ - Stage 2: WFS&C exposures use the same :ref:`calwebb_image2 <calwebb_image2>`
+   pipeline processing and steps as regular images. CRDS reftype
+   ``pars-image2pipeline`` has a specific step parameter reference for WFS&C
+   processing. The processing is identical with other image processing except
+   for the omission of the :ref:`resample <resample_step>` step.
 
  - Stage 3: The ASN generator identifies pairs of dithered WFS&C images to be
    combined via the "PATTTYPE" keyword value "WFSC". The resulting ASN
    uses the :ref:`calwebb_wfs-image3 <calwebb_wfs-image3>` pipeline
    for stage 3 processing. This pipeline consists of the single step
    :ref:`wfs_combine <wfs_combine_step>`.
+
+Configuration File Deprecation
+==============================
+
+Up to version 1.1.1, the primary way specific pipelines were referred to was by
+their configuration file name, i.e. ``calwebb_detector1.cfg``. These
+configuration files were delivered as part of the JWST calibration package.
+Below is the table that matched configuration file to observing mode it was
+intended to be used with.
+
+Post-1.1.1, configuration files are no longer the primary identifier of
+pipelines. Instead, pipelines are identified by their full class name, i.e.
+``jwst.pipeline.Detector1Pipeline``, or by their simple name, or alias, i.e.
+``calwebb_detector1``. How a pipeline is run is determined by the input data and
+what step parameter reference file in CRDS is selected by that data. The reftype
+for each pipeline, or step, is determined by appending the class name of the
+step to the string ``pars-``. For example, the reftype for
+``jwst.pipeline.Detector1Pipeline`` is ``pars-detector1pipeline``. Which
+specific reference file for a reftype is then determined by the data, just as
+with any other reference file.
+
+As a result, there are a few pipelines that no longer exist explicitly by name,
```

Comment:
"there are a few pipeline configurations that no longer exist" ?  We haven't gotten rid of any actual pipelines.

Reply:
No, no actual pipelines have been removed; they never existed. For example, the "pipeline" referred to as "calwebb_wfs-image2.cfg" was never a pipeline. It is just a specific configuration for Image2Pipeline. There was never a class `jwst.pipeline.WfsSpec2Pipeline` or any other code defining a "calwebb_wfs-image2" pipeline.

As such, as we move away from using config files and depending on CRDS, there is no `strun calwebb_wfs-image2" command. It will simply be `strun calwebb_image2` and the data and CRDS selection will choose the right configuration.

At the moment, the configuration file `calwebb_wfs-image2.cfg` is still in the package, and will appear when `collect_pipeline_cfgs` is run. However, the file, as all other CFG files, are now devoid of any actual settings; it is all CRDS-dependent.